x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings/latest_findings_table.test.tsx

```diff
@@ -108,4 +112,53 @@ describe('<FindingsTable />', () => {
       expect(screen.getByText(item.rule.name)).toBeInTheDocument();
     });
   });
+
+  it('adds filter with a cell button click', () => {
+    const names = chance.unique(chance.sentence, 10);
+    const data = names.map(getFakeFindings);
+
+    const props: TableProps = {
+      loading: false,
+      items: data,
+      sorting: { sort: { field: '@timestamp', direction: 'desc' } },
+      pagination: { pageSize: 10, pageIndex: 1, totalItemCount: 0 },
+      setTableOptions: jest.fn(),
+      onAddFilter: jest.fn(),
+    };
+
+    render(
+      <TestProvider>
+        <FindingsTable {...props} />
+      </TestProvider>
+    );
+
+    const row = data[0];
+
+    const columns = getFindingsColumns({ onAddFilter: props.onAddFilter });
+
+    columns.forEach((column) => {
+      const cellElement = screen.getByTestId(
+        TEST_SUBJECTS.getFindingsTableCellTestId(column.field, row.resource.id)
+      );
+      userEvent.hover(cellElement);
```

Comment:
tried checking for `not.isVisible()` on `cellElement` before triggering `hover` but kept getting errors about `css` from `emotion` so i dropped it. 

tips/advise welcome